src/pages/drivers/[driver].tsx

```diff
@@ -99,15 +194,59 @@ export default function Driver() {
       </p>
     )
 
-  console.log(driverQuery.data)
-  console.log(resultsQuery.data)
+    const wins = getDriverStats(
+      resultsQuery.data,
```

Comment:
Given that `resultsQuery.data` is used a few times, you could consider destructuring it: 
```js
const { data } = resultsQuery
```